en/docs/adv_examples/resnet.md

```diff
@@ -1,373 +1,353 @@
-
-如果需要在完整的imagenet2012上进行训练，则需要参考下面【进阶 Advanced】部分，我们详细介绍了数据集制作的过程，使得数据集的准备不再麻烦。
-
-准备好数据集，并在train.sh中设置好相应参数后，在命令行执行：
-
-```
-sh train.sh
+```shell
+sh evaluate.sh
 ```
 
-即可开始训练，若在屏幕上不断打印出类似下面的信息，则表明训练过程正常运行：
+The accuracy of the trained model on validation dataset with 50000 images can be obtained:
 
+```shell
+Time stamp: 2020-07-27-09:28:28
+Restoring model from resnet_v15_of_best_model_val_top1_77318.
+I0727 09:28:28.773988162    8411 ev_epoll_linux.c:82]        Use of signals is disabled. Epoll engine will not be used
+Loading data from /dataset/ImageNet/ofrecord/validation
+validation: epoch 0, iter 195, top_1: 0.773277, top_k: 0.936058, samples/s: 1578.325
+validation: epoch 0, iter 195, top_1: 0.773237, top_k: 0.936078, samples/s: 1692.303
+validation: epoch 0, iter 195, top_1: 0.773297, top_k: 0.936018, samples/s: 1686.896
 ```
-train: epoch 0, iter 200, loss: 7.024337, top_1: 0.000957, top_k: 0.005313, samples/s: 964.656
-train: epoch 0, iter 400, loss: 6.849526, top_1: 0.003594, top_k: 0.012969, samples/s: 991.474
-... train: epoch 0, iter 5000, loss: 5.557458, top_1: 0.064590, top_k: 0.174648, samples/s: 935.390
-Saving model to ./output/snapshots/model_save-20200629223546/snapshot_epoch_0. validation: epoch 0, iter 100, top_1: 0.074620, top_k: 0.194120, samples/s: 2014.683
-```
-
-可以看到：
-
-- 随着训练的进行，loss不断下降，而训练的top_1/top_k准确率不断提高（其中top_k默认为top_5准确率，可自定义）。
-- 每个epoch结束时，会做另外两个工作：1）执行一次验证，并打印出验证集上的top_1/top_k准确率；2）保存模型。
-- samples/s 用来指示训练/验证的执行速度，即每秒钟能处理的图片数量。
 
+> Before executing `sh evaluate.sh`, make sure you have prepared the validation dataset of ImageNet 2012. Please refer to [OneFlow-Benchmark](https://github.com/Oneflow-Inc/OneFlow-Benchmark/blob/master/Classification/cnns) repository to learn how to make validation dataset. 
 
+From the evaluation results of the three rounds, out model has achieved 77.32+% Top1 accuracy. 
 
-**复现实验的说明：**
+Finally, congratulations! You complete the training / validating, inference and evaluation of ResNet model on ImageNet dataset. Applause for yourself!
 
-> Q1. 多久能够完成训练？
-> 
-> 在GPU环境下，使用单机8卡（NVIDIA TITAN V），完成90个epoch的完整训练过程，大概需要15小时。
-> 
-> 中，提供ResNet50 v1.5的OneFlow实现。我们在ImageNet-2012数据集上训练90轮后，验证集上的准确率能够达到：77.318%(top1)，93.622%(top5)。
-> 
-> 训练集：80.57%（top1）
-> 
-> 验证集：77.318%（top1），93.622%（top5）
 
 
+## Details
 
-## 更详细的说明 Details
+### Distributed training
 
-### 分布式训练
+**Simple and easy-to-use distributed training is one of OneFlow's main features**
 
-**简单而易用的分布式，是OneFlow的主打特色之一。**
+OneFlow is designed to support efficient distributed training natively. Especially for distributed data parallelism, user do not have to worry about how to divide and synchronize the data when the algorithm expands from single machine to multiple machines. That is to say, in OneFlow, User only need to write algorithm from the view of single machine, and the code automatically has the ability of distributed training. 
```

Comment:
when the algorithm expands from single machine to multiple machines.
->
when the algorithm is extended from single machine to multiple machines.